app/components/search/SearchResults/SearchResults.module.scss

```diff
@@ -54,7 +54,7 @@
   display: grid;
   grid-template-columns: 7fr 3fr;
   gap: 25px;
-  justify-items: end;
+  justify-items: stretch;
```

Comment:
This fixes a bug where certain search results would return and be justified towards the map (when the text was minimal and thus the search results body was narrow to move to the end)